package.json

```diff
@@ -63,7 +63,7 @@
     "webpack-dev-server": "^2.4.0"
   },
   "engines": {
-    "node": ">= 4",
+    "node": ">= 4 <= 8",
```

Comment:
What's wrong with higher versions?

Reply:
It looks like `fsevents` 1.1.2 does not support node 9 and newer, but 1.1.3 does. I'll open a separate PR getting the appropriate packages upgraded.

Reply:
Ok I have this working in https://github.com/uber-web/mjolnir.js/pull/27, I'll remove this change from this PR.